UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs

```diff
@@ -338,11 +341,17 @@ public void SetSpriteVariant(int spriteVariant, bool networked = true)
 
 	public Color GetColor()
 	{
+		Init();
 		if (setColour == null)
 		{
 			UpdateImageColor();
 		}
 
+		if (setColour == null)
+		{
+			Loggy.LogError("AAAAAAAAAAAA");
```

Comment:
AAAAAAAAAAAAAAAAAAAAAAAA is not an informative error log.
also follow our logging standards.

Reply:
oops is debug